acquisitions/src/main/resources/thunderjet/mod-invoice/features/check-vendor-address-included-with-batch-voucher.feature

```diff
@@ -112,7 +112,7 @@ Feature: Check vendor address included with batch voucher
     # ============= get export later to give it time to create the batch voucher ===================
     Given path 'batch-voucher/batch-voucher-exports', batchVoucherExportId
     And headers headersUser
-    And retry until response.status == 'Generated'
+    And retry until response.status == 'Error'
```

Comment:
@grabsefx what is the sense to check that export failed? especially when we need XML